### PR TITLE
Fix translate 27102016

### PR DESCRIPTION
**Pull request by minhnghivn**

### CODE REVIEW

File: cdx_vietnam/config/locales/select.vi.yml [hunk 1]
```diff
@@ -127,5 +123,25 @@ vi:
...
+        instant: lập tức
+        aggregate: Tổng hợp
+        targeted: Chỉ tiêu
+      frequency_value:
+        hourly: Hàng giờ
+        daily: Hàng ngày
+        weekly: Hàng tuần
+        monthly: Hàng tháng
+
+ 		etbval: 'Etb Values Action'
```
Comment (jackregnart) [inline cdx_vietnam/config/locales/select.vi.yml]: What's going on here?

Comment (tripledub) [inline cdx_vietnam/config/locales/select.vi.yml]: Looks like some bad tabs, I guess it might stop yaml parsing correctly?

Comment (tripledub) [inline cdx_vietnam/config/locales/select.vi.yml]: Looks like some bad tabs, I guess it might stop yaml parsing correctly? Please fix before I merge this

**Comment by tripledub** [comment]
@minhnghivn I just fixed this, not need to push another version. It was tab characters, please make sure your editor is set to convert tabs to spaces (2 spaces per tab)

**Pull request merged**